app/assets/stylesheets/components/_signup-link.scss

```diff
@@ -0,0 +1,34 @@
+.app-c-signup-link--with-background-and-border {
+  padding: govuk-spacing(6);
+  background-color: govuk-colour("light-grey", $legacy: "grey-4");
+  border: 1px solid $govuk-border-colour;
+}
+
+.app-c-signup-link__link {
+  @include govuk-font(19);
+}
+
+.app-c-signup-link--link-only .app-c-signup-link__link {
+  font-weight: bold;
+  display: inline-block;
+  margin-left: govuk-spacing(5);
+  margin-bottom: 0;
+}
+
+.app-c-signup-link__inner {
+  position: relative;
+}
+
+.app-c-signup-link__icon {
+  position: absolute;
+  top: 2px;
+
+  @include govuk-media-query($until: tablet) {
+    top: 0;
```

Comment:
Could this be rewritten to use `$from:` instead? I think we should be doing this mobile first if possible.